app/models/bookmark.rb

```diff
@@ -172,8 +179,6 @@ def self.cleanup!
 #  auto_delete_preference :integer          default(0), not null
 #  pinned                 :boolean          default(FALSE)
 #  for_topic              :boolean          default(FALSE), not null
-#  bookmarkable_id        :integer
-#  bookmarkable_type      :string
```

Comment:
I think we need to run annotations again as the index for these columns is still listed below.

Reply:
The index will disappear when I drop the columns and I will re-annotate then :+1: